CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+- `selectedSeller` added as a prop
```

Comment:
Missing change type section. Please check the https://keepachangelog.com/en/1.0.0/  documentation